src/nav/accounts.yml

```diff
@@ -140,6 +140,8 @@ pages:
         path: /docs/accounts/accounts-billing/new-relic-one-pricing-billing/eligibility-guidelines-new-relic-nonprofit-program
       - title: Downgrade or cancel account
         path: /docs/accounts/accounts-billing/account-setup/downgradecancel-account
+      - title: Product promotions
+        path: /docs/accounts/accounts/account-maintenance/promotions-ui
```

Comment:
@rhetoric101 Consider renaming this to /docs/accounts/accounts/account-maintenance/previews-and-trials

Reply:
@ally-sassman previews and trials are each a different type of promotion. Alas, we don't support "trials" at the moment, but to prepare for the future, I'd like to stick with promotions because that's the word our SME uses to describe both previews and trials. Eventually, the UI and our screenshots will change to use "Promotions."